components/footer/HelpChat.js

```diff
@@ -8,64 +8,57 @@
  */
 
 // dependencies
+import { useState } from 'react';
 import { EmojiLaughing, CaretLeftFill } from 'react-bootstrap-icons';
 // local files
+import styles from './HelpChat.module.css';
+import ChatWindow from './ChatWindow';
 
 /**
- * @description - Help Chat icon and button that opens a chat window on click
+ * @description - Help Chat icon. Opens a chat window on click
+ * @param {string} props.helpChatText - chat text to display on screens > small (576px)
+ * @param {string} props.helpChatTextSmallScreen - chat text to display on screens < small (576px)
  * @return {jsx} - the HelpChat component to render
  */
-const HelpChat = () => {
+const HelpChat = ({ helpChatText, helpChatTextSmallScreen }) => {
+  // state
+  const [open, setOpen] = useState(false);
+
+  // event handlers
+  /**
+   * @description - on click event listener that toggles "open" state between true or false boolean values
+   * @return {undefined} - Nothing to return
```

Comment:
remove this line - don't need to include a `@return` comment in function description if it doesn't return anything

Reply:
Done